tests/fixtures-valid.js

```diff
@@ -109,95 +112,69 @@ function handleFixtureFile(manKey, fixKey) {
 
   const filepath = path.join(fixturePath, filename);
 
-  promises.push(new Promise((resolve, reject) => {
-    fs.readFile(filepath, `utf8`, (readError, data) => {
-      if (readError) {
-        result.errors.push(getErrorString(`File could not be read.`, readError));
-        return resolve(result);
-      }
-
-      let fixtureJson;
-      try {
-        fixtureJson = JSON.parse(data);
-      }
-      catch (parseError) {
-        result.errors.push(getErrorString(`File could not be parsed as JSON.`, parseError));
-        return resolve(result);
-      }
-
-      try {
-        // checkFixture(..) returns { errors: [..], warnings: [..] }
-        Object.assign(result, checkFixture(manKey, fixKey, fixtureJson, uniqueValues));
-      }
-      catch (validateError) {
-        result.errors.push(getErrorString(`Fixture could not be validated.`, validateError));
-      }
-      return resolve(result);
-    });
-  }));
+  try {
+    const data = fs.readFileSync(filepath, `utf8`);
```

Comment:
Can you use the promisified async version (like below) please?  
(Keep the `utf8` encoding though; you don't want it to return a raw buffer.)

https://github.com/OpenLightingProject/open-fixture-library/blob/205ed1a731037aa71bc54959852f04a4e37caeba/cli/import-fixture.js#L6

https://github.com/OpenLightingProject/open-fixture-library/blob/205ed1a731037aa71bc54959852f04a4e37caeba/cli/import-fixture.js#L33